fbrp/src/fbrp/cmd/up.py

```diff
@@ -43,6 +46,44 @@ def get_proc_names(proc_names, include_deps):
     return proc_names
 
 
+def down_existing(args: argparse.Namespace, names: typing.List[str]):
+    def find_active_proc(system_state):
+        return [
+            name
+            for name in names
+            if name in system_state.procs
+            and system_state.procs[name].state != life_cycle.State.STOPPED
```

Comment:
What about procs that are in `STOPPING` state ? Are they considered active or not ?

Reply:
We still want to wait if it's `STOPPING`. There's nothing wrong with setting the `Ask` to be `DOWN`, for something already there.